recipes/agox/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "agox" %}
+{% set version = "3.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/agox-{{ version }}.tar.gz
+  sha256: 887a940876bd8354fdfb5f90d7b8766282a063086c7edca4c1de5927e154534c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - cython
+    - cymem
+    - numpy <1.24,>=1.18
+    - pip
+  run:
+    - ase
+    - python
+    - cymem
+    - h5py
+    - matscipy
+    - scikit-learn
+    - numpy <1.24,>=1.18
```

Make it compatible with the numpy used during the build.
```suggestion
    - numpy <1.24
    - {{ pin_compatible('numpy') }}
```